config/application.yml.default

```diff
@@ -24,6 +24,7 @@ all_redirect_uris_cache_duration_minutes: 2
 allowed_ialmax_providers: '[]'
 allowed_verified_within_providers: '[]'
 account_reset_token_valid_for_days: 1
+account_reset_fraud_user_wait_period_days: 0
```

Comment:
I wonder if we should take advantage of the `allow_nil` support in `IdentityConfig`, since when I see `0` here, it worries me that we might not have _any_ delay for fraud-pending account resets (even though I see that's not it's implemented). `nil` could maybe work better for the "feature not active" logic?

Reply:
I can do nil instead.